### PR TITLE
[CoreView] Implement _getMangaFromURI

### LINKED ISSUE #2376: [CoreView] Getting weird title when URI method is used
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
No

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

**Describe the bug**
Get weird title when we use URI (copy-paste) method with coreview template sites like Shonenjumpplus

**To Reproduce**
1. Copy this URL: https://shonenjumpplus.com/episode/13933686331713121780
2. Paste it in HakuNeko
3. See the name (it should be something like Manga#970)

**Screenshots**
ffffff

**HakuNeko (please complete the following information):**
 - Version Nightly (master)

### PR DESCRIPTION
**Pull request by Robonau**
added _getMangaFromURI to fix the uri title being weird
resolves #2376

### CODE REVIEW

File: src/web/mjs/connectors/templates/CoreView.mjs [hunk 1]
```diff
@@ -18,13 +19,23 @@ export default class CoreView extends Connector {
         // The query to retrieve the single manga title from inside of this.queryManga
         this.queryMangaTitle = 'h2.series-list-title';
 
+        this.queryMangaTitleFromURI = 'h1.series-header-title';
```
Comment (27136773) [inline src/web/mjs/connectors/templates/CoreView.mjs:22]: Does this work for all websites based on CoreView or do some of them need to overwrite this?

Comment (Robonau) [reply src/web/mjs/connectors/templates/CoreView.mjs:22]: works for all the sites that don't already have it over written

Comment (Robonau) [reply src/web/mjs/connectors/templates/CoreView.mjs:22]: i tested all the connectors that use coreview

**Review by 27136773 — APPROVED** [review]
Looks good

**Pull request merged**